sasmodels/models/onion.py

```diff
@@ -345,7 +345,7 @@ def profile(sld_core, radius_core, sld_solvent, n_shells,
         if fabs(A[k]) < 1.0e-16:
             # flat shell
             z.append(z_current + thickness[k])
-            rho.append(sld_out[k])
+            rho.append(sld_in[k])
```

Comment:
Maybe change doc instead of implementation?  Is there any reason to prefer sld_in over sld_out?